### PR TITLE
feat: add function getShippingRatesShipEngine

### PR DESCRIPTION
**Pull request by wildergd**

### CODE REVIEW

**Comment by LeandroTorresSicilia** [comment]
@wildergd the ci is failing here

File: functions/getShippingRatesFromShipEngine/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const functions = require('firebase-functions');
+const calculateShippingRates = require('./services/calculateShippingRates');
+const getConfig = require('./services/getConfig');
+
+const getShippingRatesFromShipEngine = async (data) => {
+    const { shipment, shipmentId } = data;
+    if (!!shipment && !!shipmentId) {
+        throw new functions.https.HttpsError(
+            'failed_precondition',
```
Comment (LeandroTorresSicilia) [inline functions/getShippingRatesFromShipEngine/main.js:9]: it is failed-precondition and also here I think this is a bad request which maps to invalid-argument

**Comment by wildergd** [comment]
Yes it is caused by the the bug in @fstore/checks that is not validating the default value for json type against the schema.

**Comment by LeandroTorresSicilia** [comment]
@wildergd also it is failing the ci

**Review by LeandroTorresSicilia — APPROVED** [review]

**Pull request merged**